src/components/sections/testimonials.jsx

```diff
@@ -35,23 +62,32 @@ const Testimonials = props => {
       description={info.description}
     />
   ));
+
+  const [selected, setSelected] = useState(testimonialInfo[0]);
+
   return (
-    <div id="about" className="testimonials">
-          <h2 className="testimonials__title">Testimonials</h2>
-      <div className="testimonials__desktop">{testimonialChildren}</div>
-      {/* <div className="about__mobile">
-        <Carousel itemsToShow={1} itemPadding={[0]}>
-          {testimonialInfo.map(info => (
-            <TestimonialsItem
-              key={info.title}
-              img={info.img}
-              title={info.title}
-              description={info.description}
-            />
-          ))}
-        </Carousel>
-      </div> */}
-    </div>
+    <section id="testimonials" className="testimonials__container">
+      <h2 className="testimonials__title">Testimonials</h2>
+      <div className="testimonials__box">
+        <TestimonialsItem item={selected} />
+      </div>
+      <div className="planet_selection_container">
+        {console.log(selected.index)}
+        {selected.index === TESTIMONIAL_PLANETS.MERCURY ? <TestimonialsMercuryBright className="testimonials__planet" /> : <TestimonialsMercuryDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.MERCURY))}/> }
+        {selected.index === TESTIMONIAL_PLANETS.VENUS ? <TestimonialsVenusBright className="testimonials__planet" /> : <TestimonialsVenusDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.VENUS))}/> }
+        {selected.index === TESTIMONIAL_PLANETS.NEPTUNE ? <TestimonialsNeptuneBright className="testimonials__planet" /> : <TestimonialsNeptuneDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.NEPTUNE))}/>}
+        {selected.index === TESTIMONIAL_PLANETS.MARS ? <TestimonialsMarsBright className="testimonials__planet" /> : <TestimonialsMarsDim className="testimonials__planet" setSelected={() => setSelected(testimonialInfo.find(curr => curr.index === TESTIMONIAL_PLANETS.MARS))}/> }
```

Comment:
abstract this. can do something like `TESTIMONIALS.map(testimonial => <TestimonialsPlanet testimonial={testimonial} selected={selected.type === testimonial.type} setSelected={setSelected} />`

then in testimonialsPlanet, do a switch case or series of if-statements on which component to return.